src/main/kotlin/org/jetbrains/research/ml/coding/assistant/dataset/model/Dataset.kt

```diff
@@ -0,0 +1,41 @@
+package org.jetbrains.research.ml.coding.assistant.dataset.model
+
+/**
+ *  Model that accumulates all record for single person
+ *  Model for one .csv file
+ */
+data class DynamicSolution(
+    val records: List<DatasetRecord>
+) {
+    fun hasFinalSolution() = records.any { it.metaInfo.isFinalSolution }
+}
+
+/**
+ * Model for all solution in dataset
```

Comment:
solution -> solutions

Reply:
Model for all solution in dataset -> _Model for all solutions in the dataset for one task?_